src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -1955,6 +1955,61 @@ public function get_test_rest_availability() {
 		return $result;
 	}
 
+	/**
+	 * Test if 'file_uploads' directive in PHP.ini is turned off
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return array The test results.
+	 */
+	public function get_test_file_uploads() {
+		$result = array(
+			'label'       => __( 'File uploads is on in PHP ini' ),
+			'status'      => 'good',
+			'badge'       => array(
+				'label' => __( 'Media' ),
+				'color' => 'blue',
+			),
+			'description' => sprintf(
+				'<p>%s</p>',
+				__( 'The <code>file_uploads</code> directive in <code>php.ini</code> determines if uploading to is allowed in your WordPress.' )
```

Comment:
This needs split up so that the markup of `<code>file_uploads</code>` doesn't need translating, see for example how it is done for [`date_default_timezone_set`](https://github.com/WordPress/wordpress-develop/blob/88c91eeaa6ab8d73bb3f5b301f9a0faf41e8e3e2/src/wp-admin/includes/class-wp-site-health.php#L1083)